executor/executor.go

```diff
@@ -47,19 +47,21 @@ func New(sdAPI api.API, args []string) (Executor, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	switch spec.Format {
 	case "binary":
 		return NewBinary(spec, args[pos+1:])
 	case "habitat":
-		return nil, nil
+		return NewHabitat(spec, args[pos+1:])
 	case "docker":
 		return nil, nil
+	default:
+		return nil, nil
```

Comment:
This is my fault, but would you please return error with case docker and default.
If user upload unknown or docker command yaml file, sd-cmd occur panic.

Reply:
🙆